scripts/requirements.txt

```diff
@@ -3,12 +3,10 @@ contextily == 1.0.0
 contextlib2 == 0.6.0.post1
 fiona == 1.8.13
 geopandas == 0.6.1
-geopy == 2.0.0
-jupyter == 1.0.0              
```

Comment:
Based on your commit comment, can you please note in README/Getting Started that the latest version of jupyer should be downloaded separately?

Reply:
I re-added Jupyter notebook to the requirements.txt. Not all required packages were listed. Added (>=) to get the latest version.